libs/ledger-live-common/src/families/stellar/js-synchronization.ts

```diff
@@ -1,12 +1,11 @@
-import type { Operation } from "@ledgerhq/types-live";
-import { encodeAccountId } from "../../account";
 import type { GetAccountShape } from "../../bridge/jsHelpers";
 import { makeScanAccounts, makeSync, mergeOps } from "../../bridge/jsHelpers";
 import { fetchAccount, fetchOperations } from "./api";
 import { buildSubAccounts } from "./tokens";
-import { inferSubOperations } from "../../account";
+import { inferSubOperations, encodeAccountId } from "@ledgerhq/coin-framework/account/index";
 import { STELLAR_BURN_ADDRESS } from "./logic";
 import { StellarBurnAddressError } from "./errors";
+import { StellarOperation } from "./types";
 
 const getAccountShape: GetAccountShape = async (info, syncConfig) => {
```

Comment:
What could be done is to have GetAccountShape<StellarAccount> and pass the type to avoid `as` everytime we call the function

Reply:
Sorry I don't get your suggestion here?

I'd rather not change the `GetAccountShape` type here to avoid more impact in this PR, and because it's generic it only provides generic `Operation` as input.

Reply:
Sorry I meant `GetAccountShape<StellarTransaction>`

Reply:
Oh ok, yeah very good point, I agree we could use generics a lot more for different things, probably the bridges themselves too.
It's a good idea but I'd rather keep it for another PR, to not make this one even bigger.
Task created: https://ledgerhq.atlassian.net/browse/LIVE-9020